### PR TITLE
Update plugin.xml and plugin-specific services to actual IntelliJ API

### PR DESCRIPTION
**Pull request by Vassiliy-Kudryashov**
## Description

There were issues with uploading[ UnitTestBot plugin 2023.10](https://github.com/UnitTestBot/UTBotJava/actions/runs/6416856332) to the marketplace:

> Upload Not Possible
> 1 Error
> Your plugin file contains 1 error. To upload your plugin to Marketplace, please fix these errors and try again.
> 
> Service preloading is deprecated in the <com.intellij.projectService> element. Consider removing the 'preload' attribute and migrating to listeners, see https://plugins.jetbrains.com/docs/intellij/plugin-listeners.html
> 7 Warnings
> We also recommend that you fix the 7 warnings flagged in your plugin file, as this will drastically improve the quality of your plugin.
> 
> Fixing warnings is optional and will not prevent you from uploading your plugin to Marketplace.
> 
> Plugin ID specified in plugin.xml should not contain 'intellij'
> Configuration file 'withJava.xml' for optional dependency 'com.intellij.modules.java' is invalid: Invalid plugin descriptor 'withJava.xml': Unable to read plugin descriptor
> Configuration file 'withKotlin.xml' for optional dependency 'org.jetbrains.kotlin' is invalid: Invalid plugin descriptor 'withKotlin.xml': Unable to read plugin descriptor
> Configuration file 'withPython.xml' for optional dependency 'com.intellij.modules.python' is invalid: Invalid plugin descriptor 'withPython.xml': Unable to read plugin descriptor
> Configuration file 'withGo.xml' for optional dependency 'org.jetbrains.plugins.go' is invalid: Invalid plugin descriptor 'withGo.xml': Unable to read plugin descriptor
> Configuration file 'withAndroid.xml' for optional dependency 'org.jetbrains.android' is invalid: Invalid plugin descriptor 'withAndroid.xml': Unable to read plugin descriptor
> Configuration file 'withIdeaMaven.xml' for optional dependency 'org.jetbrains.idea.maven' is invalid: Invalid plugin descriptor 'withIdeaMaven.xml': Unable to read plugin descriptor
> 
Specified error was fixed, we use _annotation_ in code instead of _projectService_ tag.

## Self-check list

- [x] I've set the proper **labels** for my PR (at least, for category and component).
- [x] PR **title** and **description** are clear and intelligible.
- [ ] I've added enough **comments** to my code, particularly in hard-to-understand areas.
- [ ] The functionality I've repaired, changed or added is covered with **automated tests**.
- [ ] **Manual tests** have been provided optionally.
- [ ] The **documentation** for the functionality I've been working on is up-to-date.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -110,6 +110,10 @@ allprojects {
                 "--add-opens", "java.base/jdk.internal.vm.annotation=ALL-UNNAMED"
             )
 
+            withType<Jar> {
```
Comment (denis-fokin) [inline build.gradle.kts:113]: Looks like in the new API we faced duplicates. Have you tried DuplicatesStrategy.WARN to understand what specific files have the same path?

Comment (EgorkaKulikov) [reply build.gradle.kts:113]: Previously we had some default value of DuplicateStrategy? I do not catch the reason why it should be changed and why it is safe.

Comment (Vassiliy-Kudryashov) [reply build.gradle.kts:113]: > Looks like in the new API we faced duplicates. Have you tried DuplicatesStrategy.WARN to understand what specific files have the same path?

It doesn't depend on API version, build was broken in my envorinment. DuplicatesStrategy.EXCLUDE has been already used for some time in modules to fix the same problem locally.

Comment (Vassiliy-Kudryashov) [reply build.gradle.kts:113]: > Previously we had some default value of DuplicateStrategy? I do not catch the reason why it should be changed and why it is safe.

We have explicitly specified value for modules where similar issue appeared earlier.

Comment (denis-fokin) [reply build.gradle.kts:113]: We need to evaluate specific files that duplicate but currently I do not think that it is critical. Most likely the Gradle tries to unpack the same archive from different modules and complains about the duplicates. Let's file another bug to investigate further.

**Review by denis-fokin — APPROVED** [review]

**Pull request merged**